solaris/vector/polygon.py

```diff
@@ -10,6 +10,8 @@
 import pandas as pd
 import geopandas as gpd
 from rtree.core import RTreeError
+import shutil
+import gdal
```

Comment:
Per my comment below we can probably drop the GDAL import here. I partly want to do that because if you install gdal with pip, they're going to drop support for `import gdal` and require `from osgeo import gdal`, whereas conda installs will just be `import gdal`, so it would be great to avoid this mess.